src/mbgl/gl/context.cpp

```diff
@@ -250,6 +250,7 @@ UniqueRenderbuffer Context::createRenderbuffer(const RenderbufferType type, cons
     bindRenderbuffer = renderbuffer;
     MBGL_CHECK_ERROR(
         glRenderbufferStorage(GL_RENDERBUFFER, static_cast<GLenum>(type), size.width, size.height));
+    bindRenderbuffer = 0;
```

Comment:
@kkaefer after I removed the color renderbuffer in https://github.com/mapbox/mapbox-gl-native/pull/8431/commits/051481a78456792e4aed756a2674b28fdc1db583, the line above this (`glRenderbufferStorage`) aborts with a `GL_INVALID_OPERATION` error. I have no idea why it worked fine before I removed the color buffer (with two `glRenderbufferStorage` calls) and I don't really understand why this fixes it, but…it does… o_0 
And I figured since the binding is necessary here just to initialize the renderbuffer's storage, it wouldn't be incorrect to rebind the default renderbuffer / leaving the new renderbuffer bound wasn't necessary. ??